common/crypto_utils/Makefile

```diff
@@ -12,11 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-PY_VERSION=${shell python3 --version | sed 's/Python \(3\.[0-9]\).*/\1/' | tr -d .}
+#PY_VERSION=${shell python3 --version | sed 's/Python \(3\.[0-9]\).*/\1/' | tr -d .}
+PY_VERSION=${shell python3 --version | sed 's/Python \(3\.[0-9]\).*/\1/' | cut -b 1}
```

Comment:
Were you facing any issues with `tr -d `

Reply:
Now since the crypto_utils package is independent of swig, the python tag in package name will only have "py3" as opposed to "py36" with swig. Cut is used to remove everything after 3, in the python_version.
However earlier trim was used to remove the dot between python version 3.6.

Same pattern is used to determine PY_VERSION here as well - https://github.com/hyperledger/avalon/blob/master/common/python/Makefile#L16

Reply:
Please remove commented line

Reply:
Done